Adamant/Modules/Settings/VisibleWallets/VisibleWalletsViewController.swift

```diff
@@ -94,21 +94,6 @@ final class VisibleWalletsViewController: KeyboardObservingViewController {
     }
     
     private func addObservers() {
-        for wallet in wallets {
-            let notification = wallet.walletUpdatedNotification
-            
-            NotificationCenter.default.addObserver(
-                forName: notification,
-                object: wallet,
-                queue: OperationQueue.main
-            ) { [weak self] _ in
-                MainActor.assumeIsolatedSafe {
-                    guard let self = self else { return }
-                    self.tableView.reloadData()
-                }
-            }
-        }
-        
         NotificationCenter.default.addObserver(
```

Comment:
I think it would be nice to add removal of `self` as an observer on deinit in this ViewController just as a good practice. What do you think @vovameister ?

```swift
deinit {
        NotificationCenter.default.removeObserver(self)
    }
```